integration_test/lit.cfg.py

```diff
@@ -65,6 +65,7 @@
 
 # Tweak the PATH to include the tools dir.
 llvm_config.with_environment('PATH', config.llvm_tools_dir, append_path=True)
+config.environment['PATH'] = os.pathsep.join([config.circt_tools_dir, config.environment['PATH']])
```

Comment:
Does the following work as well?
```suggestion
config.path.append(config.circt_tools_dir)
```

Reply:
TestingConfig should not have this memver?

Reply:
Never mind, I think I was looking at a very old version of lit. Just found the following in LLVM though:
https://github.com/llvm/llvm-project/blob/main/llvm/test/lit.cfg.py#L38-L39
```suggestion
# Tweak the PATH to include the tools dir.
llvm_config.with_environment("PATH", config.llvm_tools_dir, append_path=True)
```
Seems like we're in good company.